### ISSUE TITLE
After a tiling job is complete, hit a OAM Catalog endpoint to notify new tile service for imagery.

### ISSUE THREAD

**Issue by lossyrob** [hotosm/oam-server#24]
After a completion of a tile set from some OIN imagery, we need to hit a Catalog endpoint with the following information:

``` javascript
{
  "images" : [ "s3://bucket/oinimage1.tiff", ...]
  "tileService" : "http://tiles.openaerialmap.org/imageset/{z}/{y}/{z}"
}
```

@smit1678 would this allow integration with the catalog, by allowing it to update the tile service entries in the metadata?

**Comment by smit1678** [comment]
@lossyrob Right. Not sure if you caught the latest update in document for the /tms endpoint, but it's online now: http://docs.openaerialmap.org/#api-TMS-PostTms. Will need to send you a token. This work for you?

**Comment by lossyrob** [comment]
@smit1678 perfect! Can the token be static, so that we can just bake it in to the config, or will that be another call of sorts.

**Comment by smit1678** [comment]
totally static :statue_of_liberty:

**Comment by lossyrob** [comment]
:+1:, so we'll just hit that endpoint from our side when tiling is done. Thanks!
